src/util/log.d

```diff
@@ -183,6 +185,28 @@ struct Log
     }
 }
 
+// Helper to allow using named references in format strings, ie. treat %{name} equivalent to %s.
+// Avoid std.regex because it's used at CTFE.
+private string simplifyNamedReferences(string fmt)
+{
+    string result = fmt;
+
+    while (result.canFind("%{"))
+    {
+        enforce(result.find("%{").canFind("}"), fmt ~ ": terminating '}' not found");
+        result = result[0 .. result.countUntil("%{")] ~ "%s" ~ result.find("%{").find("}").dropOne;
```

Comment:
result = result[0 .. $ - result.find("%{").length] ~ ...

just to get along with `find` only.

Reply:
done